### DISCUSSION TITLE
Feature request: timer to start block timer

### DISCUSSION THREAD

**Discussion by aryamm20** [SelfControlApp/selfcontrol#714] [Ideas]
Hello,

Would it be possible include—or could someone please tell me how to go about doing it myself—a timer to _begin_ the block timer to run? For instance, if I want to block certain websites _after_ 10 minutes, is there a way to do that?

Thank you—also not sure if this is the right format/place for one of these posts, but appreciate the help.

**Comment by slambert** [comment]
@andreasgrill created something for exactly this: 

https://github.com/andreasgrill/auto-selfcontrol

**Reply by aryamm20** [reply]
Thanks for your reply; the program you linked is very helpful. Correct me if I'm wrong, though, but it doesn't actually fulfill the function I was speaking about since it seems more geared towards enabling one to _schedule_ the blocking of websites than enabling someone to, in real time, decide that they want SelfControl to turn on within 15 minutes, for example.

On another note, with regards to implementing this program specifically, am I right in thinking that I need to edit the parameters for the schedule within terminal every time? So there isn't going to be like a menu bar or other interface where I can toggle these things.
